final_task/rss_parser/rss_parser.py

```diff
@@ -0,0 +1,126 @@
+import json
+import feedparser
+import argparse
+
+version = 1.0
+
+
+def gen_entries(parsed):
```

Comment:
Here and below:
It's a good practice to add type hits in your function signatures (e.g. def gen_entires(parsed: <some type>) -> <some_type>